app/src/components/Hand.vue

```diff
@@ -11,9 +11,21 @@
       </li>
     </ul>
 
-    <p v-if="isUserTurn && isPerishPhase">
-      You are about to die ! (autoresolve in {{ resolveCountdown }})
+    <p class="PerishChoice" v-if="isUserTurn && isPerishPhase">
+      You almost died ! Luckily, you had a Resurect card. Where do you want to
+      re-insert the Perish card ?
+      <br />
+      <Button @click="$emit('insert-perish', 0)">On top</Button>
+      <Button
+        v-for="index in playerCount"
+        :key="index"
+        @click="$emit('insert-perish', index)"
+        >Position {{ index }}</Button
+      >
+      <Button @click="$emit('insert-perish', lastIndex)">On bottom</Button>
+      <Button @click="$emit('insert-perish', randomIndex)">Random</Button>
```

Comment:
We will have to rethink the "Perish Insertion" ui. The player should be able to place the card anywhere in the deck. We could use a slider+input field and keep the buttons as shortcuts.

Reply:
I think allowing the player to insert the card anywhere is overkill, and will make the UI more complex.
Usually, you will place the card in one of the first positions in the deck, to target a specific player, or at random, or on the bottom. This is also what they do in the mobile app. Keep it simple :smile: